src/elli_http.erl

```diff
@@ -262,10 +274,16 @@ do_send_file(Fd, {Offset, Length}, #req{callback={Mod, Args}} = Req, Headers) ->
 %% client is sending. If this is not the case, {@link send_bad_request/1}
 %% might reset the client connection.
 send_bad_request(Socket) ->
-    Body = <<"Bad Request">>,
-    Response = [<<"HTTP/1.1 ">>, status(400), <<"\r\n">>,
+    send_rescue_response(Socket, 400, <<"Bad Request">>).
+
+send_server_error(Socket) ->
+    send_rescue_response(Socket, 500, <<"Server Error">>).
+
+send_rescue_response(Socket, Code, Body) ->
+    Response = [<<"HTTP/1.1 ">>, status(Code), <<"\r\n">>,
```

Comment:
`1.1` should not be hard-coded here. See [`#req.version`](https://github.com/elli-lib/elli/blob/7e9cadfe3728bcada360c606c869d3361faf4aa3/include/elli.hrl#L5) and [`elli_http:version/0`](https://github.com/elli-lib/elli/blob/7e9cadfe3728bcada360c606c869d3361faf4aa3/src/elli_http.erl#L25).